doc/api/packages.md

```diff
@@ -323,6 +332,18 @@ package. It is not a strong encapsulation since a direct require of any
 absolute subpath of the package such as
 `require('/path/to/node_modules/pkg/subpath.js')` will still load `subpath.js`.
 
+All supported versions of Node.js (since 12.7.0) and modern build tools
+support the `"exports"` field. For projects using an older version of Node.js
+or a related build tool, compatibility can be achieved by including the `"main"`
+field alongside `"exports"`:
```

Comment:
this should note that they also need to point to the same path, or at least something CJS will successfully resolve.

Reply:
Added.